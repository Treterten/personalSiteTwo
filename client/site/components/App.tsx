import React from 'react';
import styles from '../styles/header.module.css';
import Landing from './Landing';
import Navbar from './Navbar';
import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
const App = () => (
    <Router>
        <Navbar />
        <Switch>
            <Route path="/aboutme">
                <p>Coming Soon</p>
            </Route>
            <Route path="/blog">
                <p>Coming Soon</p>
            </Route>
            <Route path="/projects">
                <p>Coming Soon</p>
            </Route>
            <Route path="/">
                <Landing />
            </Route>
        </Switch>
    </Router>
)

export default App;