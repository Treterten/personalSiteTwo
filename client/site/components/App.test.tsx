/* eslint-disable import/no-extraneous-dependencies */
import React from 'react';
import { shallow } from 'enzyme';

import App from './App';

describe('App', () => {
  it('should render correctly', () => {
    const component = shallow(<App />);
    expect(component).toMatchSnapshot();
    expect(component.exists()).toBeTruthy();
  });
});
