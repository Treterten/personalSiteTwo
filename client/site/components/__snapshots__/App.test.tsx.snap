// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render correctly 1`] = `
<BrowserRouter>
  <Navbar />
  <Switch>
    <Route
      path="/aboutme"
    >
      <About />
    </Route>
    <Route
      path="/blog"
    >
      <p />
    </Route>
    <Route
      path="/projects"
    >
      <p />
    </Route>
    <Route
      path="/"
    >
      <Landing />
    </Route>
  </Switch>
</BrowserRouter>
`;
